modules/jts-awt/src/main/java/org/locationtech/jts/awt/FontGlyphReader.java

```diff
@@ -1,3 +1,8 @@
+/*
```

Comment:
Seems odd to add our copyright for code that was imported

Reply:
Oh yes, I thought I removed those. Let me try again.

Reply:
Is this whole module imported?

Reply:
All java code was copied from the original jts